libpod/options.go

```diff
@@ -999,6 +1030,20 @@ func WithCtrNamespace(ns string) CtrCreateOption {
 	}
 }
 
+// WithIsPause sets the container to be a pause container. This means the container will be sometimes hidden
+// and expected to be the first container in the pod.
+func WithIsPause() CtrCreateOption {
```

Comment:
We don't ever expect this to be used outside libpod - people should not be able to mark their own containers as pause containers. Rename to `withIsPause()` to make this internal.